src/Conda.jl

```diff
@@ -289,4 +307,30 @@ function rm_channel(channel::AbstractString, env::Environment=ROOTENV)
     runconda(`config --remove channels $channel --force`, env)
 end
 
+""""
+    create(filename, env=$ROOTENV, channels=String[])
+    create(io, env=$ROOTENV, channels=String[])
```

Comment:
I still think the name here should mirror the export name.  `import_list` since we have `export_list`

Reply:
That doesn't match up to the conda commands, but sure.